flink-table/flink-sql-client/src/test/java/org/apache/flink/table/client/gateway/local/ExecutionContextTest.java

```diff
@@ -73,12 +73,12 @@ public void testExecutionConfig() throws Exception {
 
 	@Test
 	public void testCatalogs() throws Exception {
-		final String catalogName = "catalog1";
+		final String catalogName = "inmemorycatalog";
 		final ExecutionContext<?> context = createCatalogExecutionContext();
 		final TableEnvironment tableEnv = context.createEnvironmentInstance().getTableEnvironment();
 
 		assertEquals(tableEnv.getCurrentCatalog(), catalogName);
-		assertEquals(tableEnv.getCurrentDatabase(), "mydatabase");
+		assertEquals(tableEnv.getCurrentDatabase(), "test-default-database");
```

Comment:
Nit: can we be consistent on how we define the two string variables?

Reply:
reverted